docs/User-Research/Winter 2021, Round 4 Synthesis.md

```diff
@@ -0,0 +1,166 @@
+
+**We also added two sprint responsive research questions:**
+
+6. What, if any, loose ends exist for OFA MVP which might be revealed by usability testing and design auditing?
+7. What insight might regional managers and OFA staff provide to engaging STT Users in on-going user research? 
+
+---
+
+## Whom We Talked To
+**This round, we talked to the OFA DIGIT team and Regional Offices, and five participants from States that:**
+
+- Had not participated in past research
+- Submit Universe Data
+- Submit Sample Data
+- Have caseloads either less than 3,500 per month or greater than 9,500
+
+---
+
+## What We Learned (Top Insights)
+
+Included in the learnings of this round is that our participants prefered to be called "Data Analysts" over "Data Preppers", and we will use that convention from here on in.
+
+
+### Analysts value data cleansing/validation that allows them to predict whether their submission will be accepted
+>Related to research questions 3 and 5
+
+One ST data analyst discussed that he has a program from a third party that cleanses and validates the data for him before he submits it, and it “predicts whether or not a submission would be accepted.” Other analysts indicated an interest in having this ability. 
+
+**Project Imapacts**
+- We will plan an interview with the vendor of the program and a look at the software application to leverage a competitive analysis with HMDA's data cleansing process in designing our solution.`  
+- We validated the hypothesis that STTs cannot receive immediate plain language feedback on how to correct data quality errors.
+
+### A useful distinction between data cleansing and data validating
+>Related to research question 5
+
+We clarified for our internal use that there are actually two data quality issues ("cleansing" and "validating"), and the terminologies that our users, stakeholders, and our UX team use varies in talking about it. 
+
+**Project Impacts**
+- We conducted secondary research on appropriate data analysis terminology. We determined that we would call the data prep phase the “data cleansing” phase and the submission phase “data validation” phase for our team. [[1]](#footnotes)
+- We understand that “data validation” is the process of verifying that raw data has been appropriately “cleansed.”
+- We generated the hypothesis that providing support for the data cleansing phase might help reduce the number of fatal errors in the validation phase. We've earmarked this for later testing and validation efforts for STT-MVP. 
+
+
+### Holidays can impact the ease with which grantees can meet reporting deadlines
+>Related to research question 2
+
+Holidays and 45 Day transmissions sometimes don't line up and create confusion. This insight offers another factor at play within a previously uncovered grantee pain point; that due to time constraints, preparing data which comes in for the last month in a quarter is often more difficult than preparing the data for the first two months.
+
```

Comment:
Yun has suggested that we consider allowing files that cover 1-3 months (as opposed to full quarters) which is technically possible now (but would require some zero- or space-filling for some of the section files, holding the record layouts constant). want to consider changing the layout in the future stages of dev.